conjure-java-client-verifier/src/test/resources/ignored-test-cases.yml

```diff
@@ -1,14 +1,7 @@
 client:
   autoDeserialize:
-
-   receiveBooleanExample:
-   - '{"value":0}' # jackson is casting 0 -> false and 1 -> true... MapperFeature.ALLOW_COERCION_OF_SCALARS);) in 2.9 will save us
-   - '{"value":"true"}' # jackson is casting 0 -> false and 1 -> true... MapperFeature.ALLOW_COERCION_OF_SCALARS);) in 2.9 will save us
-
    receiveStringExample:
    - '{"value":8}' # jackson coerces things to other types
```

Comment:
it's interesting that this is still allowed even with ALLOW_COERCION_OF_SCALARS on